FeedStoreChallenge/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ManagedFeedImage+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Stas Lee on 11/8/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is not supposed to be subclassed, so it should be marked as `final`:
```suggestion
final class ManagedFeedImage: NSManagedObject {
```